.buildconfig-android.yml

```diff
@@ -4,58 +4,112 @@ projects:
   fxa-client-library:
     path: components/fxa-client/android
     artifactId: fxaclient
-    publishedArtifacts: [fxaclient, fxaclient-forUnitTests, fxaclient-withoutLib]
+    publications:
+      - name: fxaclient
+        type: aar
```

Comment:
This `type: aar` bit is some annoying duplication, since `gradle` is separately configured to know that an `aar` pops out. However, we need it for automation, so :(
#1310 will make this more :100:

Reply:
Yeah, that's likely the long term solution. Until then, we can live with this 👍